### ISSUE TITLE
Compilation fails

### ISSUE THREAD

**Issue by hunger** [nixpulvis/oursh#67]
Compilation fails with a nightly compiler:

```
   Compiling oursh v0.4.3
error[E0599]: no function or associated item named `with_options` found for struct `std::fs::File` in the current scope
   --> /home/dev/.cargo/registry/src/github.com-1ecc6299db9ec823/oursh-0.4.3/src/program/posix/mod.rs:220:46
    |
220 | ...                   let file = File::with_options()
    |                                        ^^^^^^^^^^^^ function or associated item not found in `std::fs::File`

error[E0599]: no function or associated item named `with_options` found for struct `std::fs::File` in the current scope
   --> /home/dev/.cargo/registry/src/github.com-1ecc6299db9ec823/oursh-0.4.3/src/program/posix/mod.rs:229:46
    |
229 | ...                   let file = File::with_options()
    |                                        ^^^^^^^^^^^^ function or associated item not found in `std::fs::File`

error[E0599]: no function or associated item named `with_options` found for struct `std::fs::File` in the current scope
   --> /home/dev/.cargo/registry/src/github.com-1ecc6299db9ec823/oursh-0.4.3/src/program/posix/mod.rs:238:46
    |
238 | ...                   let file = File::with_options()
    |                                        ^^^^^^^^^^^^ function or associated item not found in `std::fs::File`

For more information about this error, try `rustc --explain E0599`.
error: failed to compile `oursh v0.4.3`, intermediate artifacts can be found at `/home/dev/.cache/tmp/cargo-installDjSFLz`

Caused by:
  could not compile `oursh` due to 3 previous errors
```

**Comment by nixpulvis** [comment]
Hmmmm, this should be a really small fix. But there seems like there might be another bug here too...

**Comment by nixpulvis** [comment]
:yawning_face:

Closed with 41978f5b.

**Comment by nixpulvis** [comment]
FWIW, the other bug is related to #26.
